package.json

```diff
@@ -33,5 +33,13 @@
   "dependencies": {
     "@changesets/cli": "^2.26.2",
     "mprocs": "^0.7.1"
+  },
+  "pnpm": {
+    "overrides": {
+      "@react-native/dev-middleware": "0.76.1"
+    },
+    "patchedDependencies": {
+      "@react-native/community-cli-plugin": "patches/@react-native__community-cli-plugin.patch"
```

Comment:
would be cool to contribute this to RN core

Reply:
thats the plan 👍